### PR TITLE
dev_Nickttrps

### PR DESCRIPTION
**Pull request by nickttrps**
This is my feature eieieiei.

### CODE REVIEW

File: lab-exercise-1/NickFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class NickFeature {
+      public void feature() {
+          // this is a new feature
```
Comment (nickttrps) [inline lab-exercise-1/NickFeature.java:3]: sawadee ja

Comment (sRubyy) [inline lab-exercise-1/NickFeature.java:3]: good job bro

File: lab-exercise-1/NickFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+public class NickFeature {
```
Comment (plphxwxt) [inline lab-exercise-1/NickFeature.java:1]: Your mother give you nice name na. This feauture looker cooler.

Comment (YunduoLuo) [inline lab-exercise-1/NickFeature.java:1]: nice work dude:)

File: lab-exercise-1/NickFeature.java [hunk 3]
```diff
@@ -1,5 +1,7 @@
 public class NickFeature {
       public void feature() {
           // this is a new feature
+          int x = 0;
+          x = x + 1;
```
Comment (sRubyy) [inline lab-exercise-1/NickFeature.java:5]: better one well done

Comment (tnktian) [inline lab-exercise-1/NickFeature.java:5]: very good kung .completed!

**Review by YunduoLuo — COMMENTED** [review]
niceeeeeeeee.

File: lab-exercise-1/NickFeature.java [hunk 4]
```diff
@@ -1,5 +1,7 @@
 public class NickFeature {
       public void feature() {
           // this is a new feature
```
Comment (arthasem) [inline lab-exercise-1/NickFeature.java:3]: good jjob bruh

**Pull request merged**